src/model/orders/Order.java

```diff
@@ -0,0 +1,72 @@
+package model.orders;
+
+import model.Product;
+import model.persons.Client;
+import java.util.HashMap;
+import java.util.Map;
+
+import java.util.Collection;
+
+/**
+ * The Order Class has this Client and the list of OrderDetail (list of product and quantity)
+ */
+public class Order {
+    Integer id;
+    Client client;
+    Collection<OrderDetail> orderDetailList;
+    String note;
+
+    /**
+     * @return The price of the list of products
+     */
+    protected Double getProductListPrice(){
+        Double totalPrice = 0.0;
+        for (OrderDetail orderDetail: orderDetailList) {
+            totalPrice+=orderDetail.getProduct().getPrice();
+        }
+        return totalPrice;
+    }
+
+    /**
+     * @return this method calculates the estimated preparation time of the whole order
```

Comment:
What is this for? It's not calculating anything, don't add code that you are not going to use right now.

Reply:
I'll fix it